packages/astro/client.d.ts

```diff
@@ -57,7 +57,7 @@ declare module 'astro:assets' {
 	};
 
 	type WithRequired<T, K extends keyof T> = T & { [P in K]-?: T[P] };
-	type Simplify<T> = { [KeyType in keyof T]: T[KeyType] };
+	type Simplify<T> = { [KeyType in keyof T]: T[KeyType] } & {};
```

Comment:
Just a small fix, eventually when we refactor types, we should probably have a "internal-type-utils" file somewhere with this ~~and all the other nonsense I add~~